Add two common laptop sizes

clarkbw opened the pull request:
I found 1366 x 768 and 1920 x 1080 are the most common desktop sizes according to stats counter.  
For more information see [bug 1247302 comment 6](https://bugzilla.mozilla.org/show_bug.cgi?id=1247302#c6)

clarkbw committed:
add two more common laptops

I found 1366 x 768 and 1920 x 1080 are the most common desktop sizes
according to stats counter.  For more see bug 1247302 :
https://bugzilla.mozilla.org/show_bug.cgi?id=1247302#c6

jankeromnes commented:
Thanks for checking with the stats and adding these two laptops! It makes sense to me, but I wonder if we shouldn't replace our current "Laptop with HiDPI screen" and "Laptop with MDPI screen" entries with these more common sizes. @clarkbw, do you agree?

clarkbw commented:
> replace

Yes, I can do that.  I like the touch screen one so I would leave that one but maybe put it at a similar size. Is the naming format good for you?

clarkbw committed:
remove old laptops, update touch laptop

remove HiDPI and MDPI Laptops
update the touch laptop to use 1920 x 1080 sizing

clarkbw commented:
Ok @jankeromnes I removed the HiDPI and MDPI laptops, and updated the touch laptop sizing.  I also rebased and force pushed the branch so it should just merge for you.

jankeromnes commented:
Works for me, thanks a lot for updating these!

P.S. No need to rebase or care about clean merges, I usually cherry-pick the commits directly.
